Enigmatry.Entry.Blueprint.Api.Tests/Infrastructure/ApiWebApplicationFactory.cs

```diff
@@ -59,8 +58,9 @@ protected override IHost CreateHost(IHostBuilder builder)
         return base.CreateHost(builder);
     }
 
-    private static void ConfigureContainer(ContainerBuilder builder) =>
-
-        // in the api tests we need to replace current user with TestUser
-        builder.RegisterModule(new TestModule(true)); // this allows certain components to be overriden
+    private static void ConfigureContainer(ContainerBuilder builder)
+    {
+        builder.RegisterModule<IdentityModule<TestUserProvider>>();
```

Comment:
Fixes another problem in the blueprint (i.e. resolving CurrenUser in the tests) which was blocking me from fixing the tests.